backend/djangoindia/api/serializers/event.py

```diff
@@ -1,6 +1,20 @@
 from rest_framework import serializers
 
-from djangoindia.db.models.event import Event, EventRegistration
+from djangoindia.db.models.event import Event, EventRegistration,Sponsor,Sponsorship
+
+
+
+class SponsorDetailsSerializer(serializers.Serializer):

```

Comment:
please read carefully and change it everywhere.
```suggestion
class SponsorDetailSerializer(serializers.Serializer):

```